src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampVerificationUtility.cs

```diff
@@ -21,16 +21,24 @@ internal static class Rfc3161TimestampVerificationUtility
 
 #if IS_DESKTOP
 
-        internal static bool ValidateSignerCertificateAgainstTimestamp(
-            X509Certificate2 signerCertificate,
-            Rfc3161TimestampTokenInfo tstInfo)
+        internal static Tuple<DateTimeOffset, DateTimeOffset> GetTimeStampLimits(Rfc3161TimestampTokenInfo tstInfo)
         {
             var tstInfoGenTime = tstInfo.Timestamp;
             var accuracyInMilliseconds = GetAccuracyInMilliseconds(tstInfo);
 
             var timestampUpperGenTime = tstInfoGenTime.AddMilliseconds(accuracyInMilliseconds);
             var timestampLowerGenTime = tstInfoGenTime.Subtract(TimeSpan.FromMilliseconds(accuracyInMilliseconds));
 
+            return new Tuple<DateTimeOffset, DateTimeOffset>(timestampUpperGenTime, timestampLowerGenTime);
```

Comment:
Would prefer not to use a tuple.  There's some overlap here with https://github.com/NuGet/NuGet.Client/pull/1861.